src/jvm/main/org/jetbrains/kotlinx/lincheck/trace/TraceReporter.kt

```diff
@@ -27,34 +26,30 @@ internal typealias Column<T> = List<T>
 @Synchronized // we should avoid concurrent executions to keep `objectNumeration` consistent
 internal fun StringBuilder.appendTrace(
     failure: LincheckFailure,
-    results: ExecutionResult,
     trace: Trace,
     exceptionStackTraces: Map<Throwable, ExceptionNumberAndStacktrace>,
```

Comment:
Do we still use `exceptionStackTraces`? 
If we assign exception numbers in `numberExceptionResults`, maybe we can get rid of it?

Reply:
We still use it for `appendExceptionsStackTraces`